loops-and-arrays.js

```diff
@@ -31,6 +35,18 @@ const numbers = [2, 4, 5, 29, 38];
 
 
 
+const numbers = [2, 4, 5, 29, 38];
+console.log(numbers);
+
+for(let i = 0; i < numbers.length; i++) {
+    if(numbers[i]%2==0)
+        numbers[i]=(numbers[i] * 2);
```

Comment:
Ik denk dat je in regels 43 en 46 geen ronde haakjes hoeft te gebruiken; in de uitwerkingen gebruikt ze Nova namelijk niet.